Composer/packages/client/src/components/BotRuntimeController/OpenWebChatButton.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import formatMessage from 'format-message';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { useRecoilValue } from 'recoil';
+
+import { BotStatus } from '../../constants';
+import { botStatusState, dispatcherState } from '../../recoilModel';
+import TelemetryClient from '../../telemetry/TelemetryClient';
+
+type OpenWebChatButtonProps = {
+  projectId: string;
+  isRoot: boolean;
```

Comment:
nit: isRootBot was a nice touch in the other component, I think you can change it here, makes more sense when you are reading the code.